gallery.html

```diff
@@ -3,9 +3,118 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="index.css">
     <title>Gallery</title>
+    <!-- <script>
+            function hello(element) {
+                var x = element.src;
+                console.log(x)
+                element.setAttribute('src', x );
+                console.log("Here There")
+            }
+            function back(element) {
+                element.setAttribute('src', x);
+            }
+       </script> -->
+       <script>
+            $(document).ready(function(){
+           $('#school').hover(function () {
+           this.src = 'ferris-poster2.jpg';
+       }, function () {
+           this.src = 'ferris-poster.jpg';
+       });
+           $('#holy').hover(function () {
+           this.src = 'black_knight.jpg';
+       }, function () {
+           this.src = 'holygrail_lg.jpg';
+       });
+       $('#marty').hover(function () {
+           this.src = 'btf2.jpg';
+       }, function () {
+           this.src = 'Back_to_the_Future.jpg';
+       });
+       $('#batman').hover(function () {
+           this.src = 'joker.jpg';
+       }, function () {
+           this.src = 'Dark_Knight.jpg';
+       });
+       $('#moonrise').hover(function () {
+           this.src = 'kingdom.jpg';
+       }, function () {
+           this.src = 'download.jpg';
+       });
+       $('#bride').hover(function () {
+           this.src = 'bride.jpg';
+       }, function () {
+           this.src = 'V1.jpg';
+       });
+       $('#star_wars').hover(function () {
+           this.src = 'wars2.jpg';
+       }, function () {
+           this.src = 'MV5BNzVlY2MwMjktM2E4OS00Y2Y3LWE3ZjctYzhkZGM3YzA1ZWM2XkEyXkFqcGdeQXVyNzkwMjQ5NzM@._V1_.jpg';
+       });
+       $('#indie').hover(function () {
+           this.src = 'raidersbg1.jpg';
+       }, function () {
+           this.src = '51K8ouYrHeL._SY445_.jpg';
+       });
+       });
```

Comment:
FYI, pretty much everything jQuery offers can be accomplished with vanilla javascript.  So you may want to try doing this in vanilla JS.